sample_project/Assets/SampleViewer/Samples/BuildingFilter/ContentBoxes.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ContentBoxes : MonoBehaviour
+{
+    [SerializeField] private GameObject disciplinePrefab;
+    [SerializeField] private GameObject categoryPrefab;
+    [SerializeField] private Transform contentTransform;
+    [SerializeField] private VerticalLayoutGroup verticalLayoutGroup;
+    public List<GameObject> contentList = new List<GameObject>();
+    private BuildingFilter filter;
+    [SerializeField] private RectTransform ScrollContent;
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        filter = FindObjectOfType<BuildingFilter>();
+        AddDisciplines(filter.DisciplineCategoryData);
+    }
+
+    // Update is called once per frame
+    private void Update()
+    {
+    }
+
+    public void AddDisciplines(List<Discipline> data)
+    {
+        // Clear existing content
+        foreach (Transform child in contentTransform)
+        {
+            Destroy(child.gameObject);
+        }
+
+        // Add new content
+        foreach (var discipline in data)
+        {
+            // Instantiate a discipline item
+            GameObject disciplineItem = Instantiate(disciplinePrefab);
+            disciplineItem.GetComponentInChildren<TextMeshProUGUI>().text = discipline.Name;
+            disciplineItem.transform.SetParent(verticalLayoutGroup.transform, false);
+            var disciplineContent = disciplineItem.GetComponent<RectTransform>();
+            contentList.Add(disciplineItem);
+            var disciplineLayout = disciplineItem.GetComponentInChildren<VerticalLayoutGroup>();
+
+            disciplineContent.sizeDelta = new Vector2(disciplineContent.sizeDelta.x, disciplineContent.sizeDelta.y + 30);
```

Comment:
Could be good to have these as const values in the class. Avoids magic numbers. Your call.
```suggestion
            disciplineContent.sizeDelta = new Vector2(disciplineContent.sizeDelta.x, disciplineContent.sizeDelta.y + yOffset);
```